amida-tech/cql-execution
# Feature/sar 96 fix filtering

# Related Tickets



- [SAR-96](https://jira.amida-tech.com/browse/SAR-96)

# How Things Worked (or Didn't) Before This PR
The expressions that were intended to filter out encounters of the same day only worked if there were only 0 or 1 encounters (which really means nothing was filtered at all) and would error out if there was more than one entry

# How Things Work Now (And How to Test)
The expression mentioned above now works on all lists
To test:
1. Because the test patient data objects for both diabetes measures is so large and hard to edit, check out branch `feature/sar-90-asthma` which uses the same function.
2. run `node exec-files/exec-medication-management-for-people-with-asthma.js` 
3. Examine the data to make sure it aligns with the results you're seeing
4. remove parts of the data to see how things change
# Readiness



1. [x] This PR passes all automated tests.
2. [x] This PR has no linting errors.
3. [x] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.
